datastore/processors_test.go

```diff
@@ -25,3 +26,34 @@ func TestProcessors(t *testing.T) {
 		}
 	})
 }
+
+func TestProcessorsBoltDB(t *testing.T) {
+	t.Run("set/get", func(t *testing.T) {
+
+		os.Setenv("WASMVISION_STORAGE_BOLTDB_FILENAME", "test-bolt.db")
```

Comment:
You could perhaps use `os.CreateTemp()` to create the DB file without having to manually delete it after the test.